test/testcafe/spec/ChallengeOktaVerifyTotp_spec.js

```diff
@@ -52,6 +52,12 @@ test
     await t.expect(await challengeOktaVerifyTOTPPageObject.getTotpLabel())
       .contains('Enter code from Okta Verify app');
 
+    // Verify links
+    await t.expect(await challengeOktaVerifyTOTPPageObject.switchAuthenticatorLinkExists()).ok();
+    await t.expect(challengeOktaVerifyTOTPPageObject.getSwitchAuthenticatorLinkText()).eql('Verify with something else');
```

Comment:
You know, this is a very good question. I don't know either why we check for both, but I have also seen it in different places so I copied the pattern. My only guess is that we do this in case there is more than one link and one of them is empty? Which is really unlikely... would it be ok if we clean this up in the jira you filed? [OKTA-402002](https://oktainc.atlassian.net/browse/OKTA-402002) (which I know is for the signout link... or I can file another jira too)